pkg/network/state.go

```diff
@@ -23,6 +24,20 @@ var (
 	_ State = &networkState{}
 )
 
+// Telemetry
+var (
+	closedConnDropped     = nettelemetry.NewStatGaugeWrapper(stateModuleName, "closed_conn_dropped", []string{}, "desc")
```

Comment:
No description for these?

Reply:
adding descriptions as I figure out what everything means